### ISSUE TITLE
OR as logic when filtering on categories

### ISSUE THREAD

**Issue by nitech** [pdaleramirez/super-filter#11]
How can I setup a superFilter with `OR` as logic? 

In this example, SuperFilter shows all products that have both productFamily 19806 and 22905. I want it to show the products that have the productFamily category 19806 **or** 22905:

```twig
craft.superFilter.setup(superFilterSetup, 
{
    filter: {
        productFamily: [19806, 22905]
    }
}) 
```

**Comment by pdaleramirez** [comment]
At the moment, it does not have that feature. OR logic is only supported for different category group

**Comment by pdaleramirez** [comment]
@nitech I released `1.2.1` for this feature more details https://github.com/pdaleramirez/super-filter/issues/14
